examples/files/hooks/useReducer.js

```diff
@@ -0,0 +1,41 @@
+import React, { useReducer } from 'react'
+
+const App = () => {
+  const reducer = (state, action) => {
```

Comment:
Should the reducer function go outside of `App`? This is going to declare a new function every render, and I'm not actually sure what the behavior of `useReducer` is in that case - but if we move it outside we won't have to consider that case.